Dual Account Script/README.md

```diff
@@ -12,14 +12,16 @@ For more information, see [here](https://docs.cyberark.com/Product-Doc/OnlineHel
 
 This script enables fully automatic configuration for the dual account feature.
 
+**_Notice:_** In Privilege Cloud Shared Services environment, only password authentication method is supported.
+
 ## Requirements
 - This scripts can run against Vault and PVWA v12.1 and up.
 - The script is used to create and configure new accounts as dual account. It does not configure existing account as dual account.
   - Verify the accounts you wish to configure as dual account does not exist in the Vault you are working with.
 
 ## Limitations
  
-This script supports only platforms with the following properties:
+This script supports only platforms (in PVWA environment) with the following properties:
```

Comment:
```suggestion
This script supports only platforms with the following properties, these properties are required in order to connect to PVWA:
```

Reply:
"platform" in this context means platform like UnixSSH or WindowsLocalAdmin.
and the properties belong to the platform.

As you can see here below, the platform is "Unix via SSH Keys" and its properties are:
Address, Username and SSH Private key (which is not supported)

![Screen Shot 2022-05-02 at 14 35 58](https://user-images.githubusercontent.com/6441803/166227494-da0c2465-c548-4b08-b8f5-3ebbce14fc9b.png)
e

Reply:
these are my notes from teams:
[25/04/2022 13:05] Hila Gross

- shared service only password is supported
- in limitation- add that this is in order to connect to PVWA itself
- the note and the usage example should indicate that it is Privilege Cloud Shared Services
- everywhere we talk about pCloud specifically need to change to Privilege Cloud Shared Services

from what I understood this should be what we need in order to connect to PVWA (bullet No.2)